tests/t085_arg_reg.py

```diff
@@ -19,7 +20,10 @@ def build(self, name, cflags='', ldflags=''):
         # cygprof doesn't support arguments now
         if cflags.find('-finstrument-functions') >= 0:
             return TestBase.TEST_SKIP
-
+        
+        if platform.machine().startswith('i686'):
+            cflags = '-march=native -m32 -mfpmath=sse -Ofast -flto -pg -O0 -D FASTCALL'
```

Comment:
Does all these flags are really needed?

Reply:
yes, for forcing to use mmx registers to gcc.

Reply:
To test passing arguments to the mmx register.